spec/mailers/candidate_mailer_referee_mails_spec.rb

```diff
@@ -85,6 +85,26 @@
       )
     end
 
+    context 'when a third reference is received but none are selected' do
+      let(:email) { mailer.send(:reference_received, reference) }
+
+      let(:application_form) { build(:application_form) }
+      let(:reference) { build(:reference, :feedback_provided, name: 'Scott Knowles', application_form: application_form) }
+      let(:second_reference) { build(:reference, :feedback_provided, name: 'William Adama', application_form: application_form) }
+      let(:third_reference) { build(:reference, :feedback_provided, name: 'Kara Thrace', application_form: application_form) }
```

Comment:
References will need to be built with the `selected_reference` factory—the `feedback_provided` trait no longer sets `selected: true`.

Reply:
This is covering a scenario in which a candidate has received 3 or more references back but hasn't selected any yet. So is this ok to remain as just `:feedback_provided` ?

Reply:
My fault, completely misread the context description! Yes, definitely—fine as is.